tests/sentry/incidents/action_handlers/test_msteams.py

```diff
@@ -74,6 +82,76 @@ def run_test(self, incident, method):
             incident, IncidentStatus(incident.status), metric_value
         )
 
+    @responses.activate
+    def test_build_incident_attachment(self):
```

Comment:
This was just missing entirely which is what I based the dynamic alert tests on in other test files so I created this